src/aspnet/4.7/windowsservercore-ltsc2016/Dockerfile

```diff
@@ -9,6 +9,7 @@ RUN Add-WindowsFeature Web-Server; `
     Add-WindowsFeature NET-Framework-45-ASPNET; `
     Add-WindowsFeature Web-Asp-Net45; `
     Remove-Item -Recurse C:\inetpub\wwwroot\*; `
+    [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12; `
```

Comment:
Should also add the SecurityProtocal setting in below "Install Roslyn compilers and ngen binaries" section